libkbfs/dir_data.go

```diff
@@ -216,23 +219,24 @@ func (dd *dirData) addEntryHelper(
 	ctx context.Context, name string, newDe DirEntry,
 	errorIfExists bool) error {
 	topBlock, _, err := dd.getter(
-		ctx, dd.tree.kmd, dd.rootBlockPointer(), dd.tree.file, blockRead)
+		ctx, dd.tree.kmd, dd.rootBlockPointer(), dd.tree.file, blockWrite)
```

Comment:
Because it was a bug before, and I only noticed it when running tests against this commit.  I just didn't bother correcting the previous PR since it wasn't in use anywhere yet.